src/otx/core/model/action_classification.py

```diff
@@ -37,21 +37,23 @@ class OTXActionClsModel(OTXModel[ActionClsBatchDataEntity, ActionClsBatchPredEnt
     def __init__(
         self,
         label_info: LabelInfoTypes,
+        input_size: tuple[int, int],
         optimizer: OptimizerCallable = DefaultOptimizerCallable,
         scheduler: LRSchedulerCallable | LRSchedulerListCallable = DefaultSchedulerCallable,
         metric: MetricCallable = MultiClassClsMetricCallable,
         torch_compile: bool = False,
     ) -> None:
-        self.image_size = (1, 1, 3, 8, 224, 224)
         self.mean = (0.0, 0.0, 0.0)
         self.std = (255.0, 255.0, 255.0)
         super().__init__(
             label_info=label_info,
+            input_size=input_size,
             optimizer=optimizer,
             scheduler=scheduler,
             metric=metric,
             torch_compile=torch_compile,
         )
+        self.input_size: tuple[int, int]
```

Comment:
Why are we showing the type of this here? Doesn't the `OTXModel` already declare `input_size` as a tuple?

Reply:
`OTXModel ` uses `tuple[int, int] | None` as type hint of `input_size`, which makes mypy error. For that reason, type of `input_size` is declared explicitly here.

Reply:
I have one more question. Task-specific `OTXModels` don't seem to have an `input_size` of `None`, nor do they have a default, so is there any reason for an `OTXModel` to have an `input_size` type of `None`?

Reply:
yes it's because some classes like `OVModel`  inherited from `OTXModel` need to set it to None.